### DISCUSSION TITLE
New reference MAPs brainstorming

### DISCUSSION THREAD

**Discussion by dbericat** [Project-MONAI/monai-deploy-app-sdk#289] [Ideas]
What are the next round of reference MAPs we should be looking at to create? Let's brainstorm. :)

Ideally, either some that industry wise there are papers or models already created in areas of active research. Or with existing challenges like BraTS (https://www.rsna.org/education/ai-resources-and-training/ai-image-challenge/brain-tumor-ai-challenge-2021).

**Comment by vikashg** [comment]
Because there is a segmentation example we should include a classification one too. I think we need to do the mednist classification. I should work on this been bit easy but I promise next meeting we will have that .

**Reply by MMelQin** [reply]
Is this going to be different from the existing MedNIST example, https://github.com/Project-MONAI/monai-deploy-app-sdk/tree/main/examples/apps/mednist_classifier_monaideploy?

**Reply by vikashg** [reply]
oh there it is no. So I take back my comment. I probably meant to make few notebooks in the tutorial section with the apps and the more I think about it. All the examples essentially serve the same purpose that there is an app but some examples to gear towards the new capabilities like dicom seg writer and series selector which you wrote and maybe have a tutorial section about it. 
I will go ahead make a notebook for one of the apps from the examples and share

**Comment by slbryson** [comment]
Will there be any issues with having MAPs that have proprietary models embedded in them added to the reference?
I assume there are several generic models that are common that exist already.

**Reply by MMelQin** [reply]
Are the "proprietary models", in this context, proprietary without a permissible license for it to be used in MONAI Deploy? If yes, then they can not be used.

If the question is about embedding the model file itself within the application source code, as static content, then yes, it can be done and has been done by users of the App SDK, just need to pay attention to the folder path at runtime.

**Comment by tomaroberts** [comment]
Echoing Selnur's discussion in MONAI Deploy WG meeting. Be great for the MAPs Zoo to cover different possible dimensions, e.g.:
- Medical imaging tasks: classification, segmentation, reconstruction, etc.
- Modality: MRI, CT, US, ECG, etc.
- Network type: UNet, 3D UNet, Cascade, Recurrent, etc, etc.

**Reply by vikashg** [reply]
There are some discussions and developments regarding model zoo and how to make it operational 
https://github.com/Project-MONAI/monai-deploy-app-sdk/discussions/230
There was an issue here https://github.com/Project-MONAI/monai-deploy-app-sdk/issues/213

**Comment by MMelQin** [comment]
I'd like to see a new app supporting model for XRay modality, either segmentation or classification, simply because we had been focusing on 3D models and DICOM CT images. In the last little while, a few defects have been found and fixed for dealing with DICOM XRay images.

Next, we can expand into creating apps for all the example models in MONAI core, as well as custom applications supporting models that were not trained with MONAI (permissible license off course).

**Reply by vikashg** [reply]
There is the stanford MURA dataset but they are all JPEG images. Let me dig into find some datasets with dicom images.
